controllers/dsc/config.go

```diff
@@ -50,7 +50,7 @@ func (r *DSCConfigReader) ReadDSCConfig(ctx context.Context, log *logr.Logger) (
 		if errors.IsNotFound(err) {
 			log.V(1).Info("DSC ConfigMap not found, using default configuration",
 				"configmap", configMapKey)
-			return &DSCConfig{}, nil
```

Comment:
shouldn't here return an error as well? 🤔 otherwise later in:

https://github.com/trustyai-explainability/trustyai-service-operator/blob/6b4663033aa7269fc07a3ea1f5c06881ec4d2211/controllers/lmes/lmevaljob_controller.go#L280-L287

in case this CM is not found, you end up again in line 286, meaning you apply `false`, `false` on top of the trustyai-service-operator-config values?

Reply:
nevermind I retract that, I see:

https://github.com/trustyai-explainability/trustyai-service-operator/blob/6b4663033aa7269fc07a3ea1f5c06881ec4d2211/controllers/lmes/config.go#L120-L126